packages/x-components/src/x-modules/next-queries/components/next-queries-list.vue

```diff
@@ -89,6 +89,15 @@
     @Prop()
     public maxGroups!: number;
 
+    /**
+     * Determines if a group is added to the injected items list when the number
+     * of items is smaller than the offset.
+     *
+     * @public
+     */
+    @Prop({ default: true })
+    public concatWhenNoMoreItems!: boolean;
```

Comment:
I found it difficult to understand what this does, even with the docs (maybe it's a "me" problem). Did you have any name alternative? `concatWehnTrailing`, `concatAtItemsListEnd` or something like that?